.github/workflows/issues-loader.yaml

```diff
@@ -42,4 +42,10 @@ jobs:
           # use the value of the dryRun input parameter; otherwise, use 'false'
           DRY_RUN: ${{ github.event_name == 'workflow_dispatch' && inputs.dryRun || 'false' }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Commit and push changes
```

Comment:
I'm guess there won't be a commit if nothing changed in the state file, right?

Reply:
Yepp. That's what I understand based on the actions' docs 😛